geode-docs/tools_modules/gfsh/command-pages/create.html.md.erb

```diff
@@ -924,22 +924,22 @@ If this option is specified without a value or is specified with a value of <cod
 </tr>
 <tr class="odd">
 <td><span class="keyword parmname">\-\-region-idle-time-expiration</span></td>
-<td>How long the region can remain in the cache without being accessed. The default is no expiration of this type.</td>
+<td>How long the region entries can remain in the cache without being accessed. The default is no expiration of this type.</td>
```

Comment:
You could say: "How long a region will remain unexpired whose entries have not been accessed"
We don't need to define here what happens when it expires since that will be done under "action".
Do the docs include what the time unit is (i.e. millis, seconds, ...). I think it should.

Reply:
What about "How long the region can remain in the cache without its entries being accessed." ?
+1 for the time units

Reply:
Expiration times are specified in seconds, according to geode/geode-core/src/main/java/org/apache/geode/cache/ExpirationAttributes.java.

Reply:
@alb3rtobr  Do you have enough information to make the next round of revisions?